opensrp-giz-malawi/src/test/java/org/smartregister/giz/activity/ReportTypeListRegisterActivityTest.java

```diff
@@ -0,0 +1,122 @@
+package org.smartregister.giz.activity;
+
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.robolectric.Robolectric;
+import org.robolectric.android.controller.ActivityController;
+import org.robolectric.util.ReflectionHelpers;
+import org.smartregister.Context;
+import org.smartregister.CoreLibrary;
+import org.smartregister.domain.FetchStatus;
+import org.smartregister.giz.BaseUnitTest;
+import org.smartregister.giz.R;
+
+public class ReportTypeListRegisterActivityTest extends BaseUnitTest {
```

Comment:
Update **all** tests to match convention.

Reply:
Done